service-tests/github.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'github', title: 'GitHub' });
+module.exports = t;
+
+t.create('downloads for release without slash')
+  .get('/downloads/atom/atom/v0.190.0/total.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('downloads'),
+    value: Joi.not('none')
```

Comment:
I have seen obviously wrong strings like `'undefined'` and `'NaN'` show up on badges. `Joi.not('none')` will allow those strings too.